### ISSUE TITLE
Compatibility with `/marker`

### ISSUE THREAD

**Issue by Uriei** [chatty/chatty#271]
Twitch added a new command for Casters and Editors a few weeks ago to add a marker for later editing of a VOD into a highlight, but when used from Chatty it answers `[Info] Sorry, "/marker" is not available through this client.`

**Comment by tduva** [comment]
As far as I know there is no official API for it, so probably not.

**Comment by Uriei** [comment]
Yeah, I understand that it might not be able to be implemented, I feel that its not just a "hey add a marker right now" and more like a "Hey, add a marker to this exact timestamp I'm watching right now" so it will probably depend on you actively watching the stream at that moment.

**Comment by marcmagus** [comment]
Stream markers now have an API. This feature would be highly desirable now that it might be possible.

https://discuss.dev.twitch.tv/t/introducing-the-stream-markers-api/17571
